tsconfig.json

```diff
@@ -18,5 +18,5 @@
   },
   "exclude": ["node_modules"],
   "files": ["./hardhat.config.ts"],
-  "include": ["tasks/**/*", "test/**/*", "types/**/*"]
+  "include": ["tasks/**/*", "test/**/*", "types/**/*", "src/**/*"]
```

Comment:
My only concern is this line of code that I don't quite understand

Reply:
typescript has to know where our .ts files are. This array that we provide to the "include" property specifies where the ts files in our project are. Before, typescript didnt know we had a src folder. Now we're telling it, "check everything inside src and do typescript magic on it"

Reply:
Ok got it